root/static/scripts/guess-case/modes.js

```diff
@@ -282,3 +291,27 @@ exports.Sentence = assign({}, DefaultMode, {
     {url: {href: 'https://musicbrainz.org/doc/Style/Language', target: '_blank'}}
   ),
 });
+
+exports.Turkish = assign({}, DefaultMode, {
+  description: l(
+    'This mode capitalises almost all words, the connectives ' +
+    'and the question suffixes used in a title are lowercased.'+
+    'In Turkish, capital of \'i\' is \'İ\' and of \'ı\' is \'I\' and'+
+    'this mode handles these. Some words, often proper nouns, ' +
+    'may need to be manually fixed according to the {url|Turkish ' +
+    'language guidelines}.',
```

Comment:
[Capital](https://en.wikipedia.org/wiki/Letter_case) seems to be correctly used. But the overall phrasing of this description can certainly be improved.